README.md

```diff
@@ -31,16 +32,29 @@
 </p>
 </br>
 
-Semgrep is a fast, open-source, static analysis tool that finds bugs and enforces code standards at editor, commit, and CI time. Precise rules look like the code you’re searching; no more traversing abstract syntax trees, wrestling with regexes, or using a painful DSL. Code analysis is performed locally (code is not uploaded) and Semgrep runs on uncompiled code.
+Semgrep is a fast, open-source, static analysis tool for finding bugs and enforcing code standards at editor, commit, and CI time. [Get started →.](#getting-started)
+
+Semgrep runs fully on your computer or build environment: **your code is never sent anywhere**.
```

Comment:
I like how clear this is, but I prefer the previous language for its specificity of where the analysis happens.

Reply:
Updated.